src/Aspire.Hosting/Dcp/DcpExecutor.cs

```diff
@@ -1735,6 +1865,13 @@ public async Task StartResourceAsync(IResourceReference resourceReference, Cance
                     await _executorEvents.PublishAsync(new OnResourceStartingContext(cancellationToken, resourceType, appResource.ModelResource, appResource.DcpResourceName)).ConfigureAwait(false);
                     await CreateExecutableAsync(appResource, resourceLogger, cancellationToken).ConfigureAwait(false);
                     break;
+                case ContainerExec containerExec:
```

Comment:
Again, since there is no way to "stop" ContainerExec (it runs automatically, to completion, after it is created), there should be no way to "start" it either.

Reply:
ah, i see. I removed `StopResourceAsync` and `StartResourceAsync` support for `ContainerExec` completely